<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MetadataWidget</class>
 <widget class="QWidget" name="MetadataWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>814</width>
    <height>525</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Adobe 黑体 Std R</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_13">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>8</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <property name="spacing">
        <number>2</number>
       </property>
       <item>
        <widget class="QPushButton" name="add_file_button">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Adobe 黑体 Std R</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>添加文件</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="delete_file_button">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Adobe 黑体 Std R</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>去除文件</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListWidget" name="file_listWidget">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_15">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_12">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Adobe 黑体 Std R</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>文件原信息</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="font">
            <font>
             <family>Adobe 黑体 Std R</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>路径</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="path_label">
           <property name="maximumSize">
            <size>
             <width>250</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QLabel" name="label_11">
           <property name="font">
            <font>
             <family>Adobe 黑体 Std R</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>文件名</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="filename_label">
           <property name="maximumSize">
            <size>
             <width>250</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <widget class="QLabel" name="label_12">
           <property name="font">
            <font>
             <family>Adobe 黑体 Std R</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>曲名</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="original_song_name_label">
           <property name="maximumSize">
            <size>
             <width>250</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_10">
         <item>
          <widget class="QLabel" name="label_14">
           <property name="font">
            <font>
             <family>Adobe 黑体 Std R</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>歌手</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="original_singer_label">
           <property name="maximumSize">
            <size>
             <width>250</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_11">
         <item>
          <widget class="QLabel" name="label_15">
           <property name="font">
            <font>
             <family>Adobe 黑体 Std R</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>时长</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="original_duration_label">
           <property name="maximumSize">
            <size>
             <width>250</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_13">
         <item>
          <widget class="QLabel" name="label_16">
           <property name="font">
            <font>
             <family>Adobe 黑体 Std R</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>md5值</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="original_md5_label">
           <property name="maximumSize">
            <size>
             <width>250</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_14">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="font">
            <font>
             <family>Adobe 黑体 Std R</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>手动修改</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pass_button">
           <property name="font">
            <font>
             <family>Adobe 黑体 Std R</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string>跳过该文件</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <item>
          <widget class="QPushButton" name="setting_button">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>34</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>设置</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="confirm_buton">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>44</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Adobe 黑体 Std R</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>导入元数据</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <property name="spacing">
      <number>4</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLineEdit" name="search_lineEdit">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>35</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="search_button">
           <property name="minimumSize">
            <size>
             <width>35</width>
             <height>35</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>35</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>o</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="search_tableWidget">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="result_pic_label">
         <property name="minimumSize">
          <size>
           <width>140</width>
           <height>140</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>140</width>
           <height>140</height>
          </size>
         </property>
         <property name="text">
          <string>N/A</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <property name="spacing">
            <number>2</number>
           </property>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="maximumSize">
              <size>
               <width>45</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="baseSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Adobe 黑体 Std R</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>时长</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="result_duration_label">
             <property name="font">
              <font>
               <family>Adobe 黑体 Std R</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>N/A</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <property name="spacing">
            <number>2</number>
           </property>
           <item>
            <widget class="QLabel" name="label_9">
             <property name="maximumSize">
              <size>
               <width>45</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="baseSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Adobe 黑体 Std R</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>年份</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="result_year_label">
             <property name="font">
              <font>
               <family>Adobe 黑体 Std R</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>N/A</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <property name="spacing">
            <number>2</number>
           </property>
           <item>
            <widget class="QLabel" name="label_8">
             <property name="maximumSize">
              <size>
               <width>45</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="baseSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Adobe 黑体 Std R</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>序号</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="result_track_number_label">
             <property name="font">
              <font>
               <family>Adobe 黑体 Std R</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>N/A</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <property name="spacing">
            <number>2</number>
           </property>
           <item>
            <widget class="QLabel" name="label_7">
             <property name="maximumSize">
              <size>
               <width>45</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="baseSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Adobe 黑体 Std R</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>流派</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="result_genre_label">
             <property name="font">
              <font>
               <family>Adobe 黑体 Std R</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>N/A</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>2</number>
       </property>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="maximumSize">
          <size>
           <width>45</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Adobe 黑体 Std R</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>曲名</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="result_song_name_label">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Adobe 黑体 Std R</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>N/A</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>4</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>2</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="maximumSize">
            <size>
             <width>45</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Adobe 黑体 Std R</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>歌手</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="result_singer_label">
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>0</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Adobe 黑体 Std R</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>N/A</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <property name="spacing">
          <number>2</number>
         </property>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="maximumSize">
            <size>
             <width>45</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Adobe 黑体 Std R</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>专辑</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="result_album_label">
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>0</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Adobe 黑体 Std R</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>N/A</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
